CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# dbt_twilio v0.2.0
+
```

Comment:
Should we also include the breaking change callout from the source package?

Reply:
added